presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/SelectorSpec.java

```diff
@@ -38,12 +39,14 @@ public SelectorSpec(
             @JsonProperty("source") Optional<Pattern> sourceRegex,
             @JsonProperty("queryType") Optional<String> queryType,
             @JsonProperty("clientTags") Optional<List<String>> clientTags,
+            @JsonProperty("resourceEstimatesRange") Optional<ResourceEstimatesRange> resourceEstimatesRange,
```

Comment:
I think we can name this as `resourceEstimate` or `resourceUsageEstimate` as I commented above. I find the latter more clear.